404 error on npm install

I am trying to use this package but when i try to npm install @contentful/contentful-slatejs-adapter i get this message.

`npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@contentful%2fcontentful-slatejs-adapter - Not found
npm ERR! 404
npm ERR! 404  '@contentful/contentful-slatejs-adapter@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404`

can anyone help me with that?

--- a/packages/contentful-slatejs-adapter/package.json
+++ b/packages/contentful-slatejs-adapter/package.json
@@ -91,7 +91,7 @@
     "typescript": "^4.2.4"
   },
   "publishConfig": {
-    "access": "restricted"
+    "access": "public"
   },
   "dependencies": {
     "@contentful/rich-text-types": "^5.0.0",

